xamarin/_Shared/realm.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+    "app_id": "todo-sync-tmikv",
```

Comment:
Did you intentionally include an appID here?

Reply:
Yes. Will be overridden by backend generation.